suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/TestFakeTripGen.java

```diff
@@ -49,4 +53,119 @@ public void testInputMasks() {
 			}
 		}
 	}
+	
+	@Test
+    public void testInputMasksTowards() {
+        int amount = 100000;
+        int noOfParties = 2;
+        int towardsPlayer = 1;
+        BigInteger modulus = new BigInteger("6703903964971298549787012499123814115273848577471136527425966013026501536706464354255445443244279389455058889493431223951165286470575994074291745908195329");
+        BigInteger alpha = new BigInteger("5081587041441179438932635098620319894716368628029284292880408086703438041331200877980213770035569812296677935118715454650749402237663859711459266577679205");
+        List<SpdzInputMask[]> inputMasks = FakeTripGen
+            .generateInputMasks(amount, towardsPlayer, noOfParties, alpha, modulus);        
+        for(SpdzInputMask[] masks : inputMasks) {
+          Assert.assertNotNull(masks[towardsPlayer-1].getRealValue());
+        }
```

Comment:
We should also test that the mask is well formed. I.e., that the shares add up to the realvalue